### ISSUE TITLE
Articles for iQuHack

### ISSUE THREAD

**Issue by HuangJunye** [HuangJunye/Qiskit-Hackathon-Guide#9]
Haven't verified: https://medium.com/mit-6-s089-intro-to-quantum-computing. After verification will add to past hackathons.

**Comment by starktech23** [comment]
Yeah I see many articles on this based on the projects made during iQuHACK. Will have to add each article (and in some cases new repos) for each individual project.
